added systemd collector config (#1120)
fluent/fluent-bit-docs: pipeline/inputs/node-exporter-metrics.md

| filesystem.ignore\_mount\_point\_regex  | Specify the regex for the mount points to prevent collection of/ignore. | `^/(dev|proc|run/credentials/.+|sys|var/lib/docker/.+|var/lib/containers/storage/.+)($|/)`    |
| filesystem.ignore\_filesystem\_type\_regex  | Specify the regex for the filesystem types to prevent collection of/ignore. | `^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$`    |
| diskstats.ignore\_device\_regex | Specify the regex for the diskstats to prevent collection of/ignore. | `^(ram|loop|fd|(h|s|v|xv)d[a-z]|nvme\\d+n\\d+p)\\d+$` |
| systemd_service_restart_metrics | Determines if the collector will include service restart metrics | false |
| systemd_unit_start_time_metrics | Determines if the collector will include unit start time metrics | false |
| systemd_include_service_task_metrics | Determines if the collector will include service task metrics | false |

> | systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | it's not applied unless explicitly set |

Reviewer: I'm (apologies for the pun) not a huge fan of contractions like `it's` particularly when this is further confused by possessive forms. Let us try to ensure we use the full set of words, it is easy to accidentally misread or typo with a contraction.
```suggestion
| systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | It is not applied unless explicitly set |
```

> | systemd_exclude_pattern | regex you'd set to determine which units are excluded in the metrics produced by the systemd collector | `.+\\.(automount|device|mount|scope|slice)"`    |

Reviewer: As above, try to remove contractions.
```suggestion
| systemd_exclude_pattern | regex to determine which units are excluded in the metrics produced by the systemd collector | `.+\\.(automount|device|mount|scope|slice)"`    |
```

| time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
| uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
| vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |

> | systemd collector | Exposes statistics fr systemd.                                                           | Linux  | v1.8.2  |

Reviewer: I think this is more recent than 1.8.2? 2.1.4?

Reviewer 2: Systemd collector was introduced in 2.1.3:
https://github.com/fluent/fluent-bit/commit/ab60c002ae9edae6548ee129f9f38cecf7732239

> | systemd collector | Exposes statistics fr systemd.                                                           | Linux  | v2.1.3  |

Reviewer: ```suggestion
| systemd collector | Exposes statistics from systemd.                                                           | Linux  | v2.1.3  |
```
